content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -99,10 +99,12 @@ limitation and compatibility rules will change.
 
 #### Pause Image
 
-Microsoft maintains a Windows pause infrastructure container at
-`mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
-Kubernetes maintains a multi-architecture image `k8s.gcr.io/pause:3.5` that
-supports Linux as well as Windows.
+Kubernetes maintains a multi-architecture image that includes support for Windows at `k8s.gcr.io/pause:3.5`.
+Source for the pause image can be found  [here](https://github.com/kubernetes/kubernetes/tree/master/build/pause).
+
+Microsoft maintains a multi-architecture image with Linux and Windows amd64 support at `mcr.microsoft.com/oss/kubernetes/pause:3.5`.
```

Comment:
Similar to the earlier change, I'm wary of specifying a particular version. Is it OK that readers should use either the 3.5 version or any later version?

Reply:
Same comments as above.